stylesheets/main.css

```diff
@@ -21,11 +37,17 @@ div#landing-header {
   padding: 0 !important;
 }
 
+div.full-vp-bg-image {
+  position: fixed;
```

Comment:
Do you prefer it this way? I like it how it is, without me sneakily creeping on the viewers as they scroll.

I think it would be interesting to make the navigation sticky or something though (in the long term, if we have more content).

Reply:
This was actually prompted from an offhand suggestion from @anitasunshine. She was inspired by the how the banner looked on https://www.wsgr.com/en/.